### ISSUE TITLE
It appears that this package doesn't work outside of Windows and MacOS ecosystem

### ISSUE THREAD

**Issue by victor-yunenko** [dreipol/linkit#6]
It relies on the assumption that the FS in question must be case-insensitive.

Here's a traceback from linux:

<details>

```
linkit==0.1.5 not in cache, need to check index
Traceback (most recent call last):
File "/root/.local/bin/pip-compile", line 10, in <module>
sys.exit(cli())
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/click/core.py", line 764, in __call__
return self.main(*args, **kwargs)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/click/core.py", line 717, in main
rv = self.invoke(ctx)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/click/core.py", line 956, in invoke
return ctx.invoke(self.callback, **ctx.params)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/click/core.py", line 555, in invoke
return callback(*args, **kwargs)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/piptools/scripts/compile.py", line 203, in cli
results = resolver.resolve(max_rounds=max_rounds)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/piptools/resolver.py", line 111, in resolve
has_changed, best_matches = self._resolve_one_round()
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/piptools/resolver.py", line 193, in _resolve_one_round
for dep in self._iter_dependencies(best_match):
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/piptools/resolver.py", line 285, in _iter_dependencies
dependencies = self.repository.get_dependencies(ireq)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/piptools/repositories/pypi.py", line 151, in get_dependencies
self._dependencies_cache[ireq] = reqset._prepare_file(self.finder, ireq)
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/pip/req/req_set.py", line 634, in _prepare_file
abstract_dist.prep_for_dist()
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/pip/req/req_set.py", line 129, in prep_for_dist
self.req_to_install.run_egg_info()
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/pip/req/req_install.py", line 439, in run_egg_info
command_desc=\'python setup.py egg_info\')
File "/root/.local/venvs/pip-tools/lib/python3.6/site-packages/pip/utils/__init__.py", line 707, in call_subprocess
% (command_desc, proc.returncode, cwd))
pip.exceptions.InstallationError: Command "python setup.py egg_info" failed with error code 1 in /tmp/tmpgsn7qpdebuild/LinkIt/
```

</details>

**Comment by victor-yunenko** [comment]
I believe @yvladislav has fixed it already - https://github.com/what-digital/linkit/commit/15f63f3471a44176bc3e637a69406da377b78186

@yvladislav could you submit a PR?

**Comment by sidneywidmer** [comment]
@victor-yunenko Thanks for your report. I'm on this and will merge it this week 👍

**Comment by victor-yunenko** [comment]
Sounds great, thank you!

**Comment by sidneywidmer** [comment]
@victor-yunenko I just released v0.1.6, let me know if everything works now for you 👍
